apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -754,16 +754,35 @@ function getGutenboardingStatus( calypsoPort ) {
 		[ port2 ]
 	);
 	port1.onmessage = ( { data } ) => {
-		const { isGutenboarding, frankenflowUrl, isNewLaunch, isExperimental } = data;
+		const {
+			isGutenboarding,
+			frankenflowUrl,
+			isNewLaunch,
+			isNewLaunchMobile,
+			isExperimental,
+		} = data;
 		calypsoifyGutenberg.isGutenboarding = isGutenboarding;
 		calypsoifyGutenberg.frankenflowUrl = frankenflowUrl;
 		calypsoifyGutenberg.isNewLaunch = isNewLaunch;
+		calypsoifyGutenberg.isNewLaunchMobile = isNewLaunchMobile;
 		calypsoifyGutenberg.isExperimental = isExperimental;
 		// Hook necessary if message recieved after editor has loaded.
 		window.wp.hooks.doAction( 'setGutenboardingStatus', isGutenboarding );
 	};
 }
 
+function handleLaunchModal( calypsoPort ) {
+	subscribe( () => {
+		const { isSidebarOpen } = select( 'automattic/launch' ).getState();
```

Comment:
This was reverted to fix an issue: https://github.com/Automattic/wp-calypso/pull/46475

In retrospect it should be clear, but this is depending on a store that is registered via a script that's registered in another plugin (editing toolkit) and nothing guarantees it will be available. The other plugin may not be active at all.

It's very fragile and risky.